cmd/use.go

```diff
@@ -21,6 +21,7 @@ import (
 	"log"
 	"os"
 
+	"github.com/manifoldco/promptui"
```

Comment:
The current build for this PR is failing in Travis ci. 
I believe it could be to do with this new package